tests/hirte_tests/gather/journal.py

```diff
@@ -1,11 +1,13 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
-import os
+from typing import List
 
```

Comment:
Do we need to have those empty lines between `from ...` imports?

Reply:
No, but I am used to group imports from other files in the projects and imports from libraries. Fixed this. Thanks!